sql/moz-fx-data-shared-prod/monitoring_derived/airflow_dag_v1/query.sql

```diff
@@ -1,13 +1,21 @@
 SELECT
   dag_id,
-  IF(is_subdag, SPLIT(dag_id, ".")[OFFSET(0)], dag_id) AS root_dag_id,
```

Comment:
why are you putting the whole thing in as root dag_id? it could be root_dag_id.subdag_id? no?